File: app/src/main/java/com/garymcgowan/moviepedia/network/OmdbBaseResponse.java
package com.garymcgowan.moviepedia.network;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Gary on 2016/10/27.
 */

public class OmdbBaseResponse {

    //error fields
//    {
//        "Response": "False",
//            "Error": "Movie not found!"
//    }

    @SerializedName("Response") boolean response;
    @SerializedName("Error") String error;


    public boolean isResponse() {
        return response;
    }

    public String getError() {
        return error;
    }
}

File: app/src/main/java/com/garymcgowan/moviepedia/network/OmdbBaseResponse.kt
package com.garymcgowan.moviepedia.network

import com.google.gson.annotations.SerializedName

open class OmdbBaseResponse {

    //error fields
    //    {
    //        "Response": "False",
    //            "Error": "Movie not found!"
    //    }

    @SerializedName("Response") var isResponse: Boolean = false
    @SerializedName("Error") var error: String? = null
}
